inc/enqueue.php

```diff
@@ -67,11 +67,11 @@ function uds_wp_metaboxes_scripts() {
 		global $pagenow;
 		$the_theme     = wp_get_theme();
 		$theme_version = $the_theme->get( 'Version' );
-		$js_metaboxes_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/metaboxes.js' );
+		$js_admin_blocks_version = $theme_version . '.' . filemtime( get_template_directory() . '/js/admin-blocks.js' );
 		$css_metaboxes_version = $theme_version . '.' . filemtime( get_template_directory() . '/css/admin.min.css' );
 		if ( 'post.php' == $pagenow || 'post-new.php' == $pagenow ) {
 
-			wp_enqueue_script( 'uds-wordpress-sidebar-metabox-scripts', get_template_directory_uri() . '/js/metaboxes.js', array( 'jquery' ), $js_metaboxes_version, true );
+			wp_enqueue_script( 'uds-wordpress-sidebar-metabox-scripts', get_template_directory_uri() . '/js/admin-blocks.js', array( 'jquery' ), $js_admin_blocks_version, true );
```

Comment:
It looks like we left the original file handle of 'uds-wordpress-sidebar-metabox-scripts' in there. While it would still work (as long as that name is unique), we should probably rename this to avoid confusion since this is not a script about meta boxes. I would suggest something like 'uds-wordpress-admin-blocks' to replace that here, and in line 75 below.